spec/feature/queue/cavc_task_queue_spec.rb

```diff
@@ -10,47 +10,113 @@
+        # Logged in as CAVC Lit Support admin
+        User.authenticate!(user: org_admin)
+        visit "queue/appeals/#{send_task.appeal.external_id}"
+
+        find(".cf-select__control", text: "Select an action").click
+        find("div", class: "cf-select__option", text: Constants.TASK_ACTIONS.ASSIGN_TO_PERSON.label).click
+
+        find(".cf-select__control", text: org_admin.full_name).click
+        find("div", class: "cf-select__option", text: org_nonadmin.full_name).click
+        fill_in "taskInstructions", with: "Confirm info and send letter to Veteran."
+        click_on "Submit"
+        expect(page).to have_content COPY::ASSIGN_TASK_SUCCESS_MESSAGE % org_nonadmin.full_name
+
+        # Logged in as first user assignee
+        User.authenticate!(user: org_nonadmin)
+        visit "queue/appeals/#{send_task.appeal.external_id}"
+
+        find(".cf-select__control", text: "Select an action").click
+        expect(page).to have_content Constants.TASK_ACTIONS.MARK_COMPLETE.label
+        expect(page).to have_content Constants.TASK_ACTIONS.REASSIGN_TO_PERSON.label
+
+        find("div", class: "cf-select__option", text: Constants.TASK_ACTIONS.REASSIGN_TO_PERSON.label).click
+        find(".cf-select__control", text: COPY::ASSIGN_WIDGET_DROPDOWN_PLACEHOLDER).click
+        find("div", class: "cf-select__option", text: org_nonadmin2.full_name).click
+        fill_in "taskInstructions", with: "Going fishing. Handing off to you."
+        click_on "Submit"
+        expect(page).to have_content COPY::REASSIGN_TASK_SUCCESS_MESSAGE % org_nonadmin2.full_name
+      end
+
+      step "assigned user completes task" do
+        # Logged in as second user assignee (due to reassignment)
+        User.authenticate!(user: org_nonadmin2)
+        visit "queue/appeals/#{send_task.appeal.external_id}"
+
+        find(".cf-select__control", text: "Select an action").click
+        find("div", class: "cf-select__option", text: Constants.TASK_ACTIONS.MARK_COMPLETE.label).click
+        fill_in "completeTaskInstructions", with: "Letter sent."
+        click_on COPY::MARK_TASK_COMPLETE_BUTTON
+        expect(page).to have_content COPY::MARK_TASK_COMPLETE_CONFIRMATION % vet_name
+
+        # Check that appeal is in correct tab in user's queue
+        find(".cf-tab", text: "Completed").click
+        expect(page).to have_content send_task.appeal.docket_number
+
+        # Check that appeal is in correct tab in Team view
+        User.authenticate!(user: org_admin)
+        visit "organizations/cavc-lit-support"
+        find(".cf-tab", text: "Assigned").click
+        expect(page).to have_content send_task.appeal.docket_number
+        # Check that org_admin has option to End hold early
+        visit "queue/appeals/#{send_task.appeal.external_id}"
+        find(".cf-select__control", text: "Select an action").click
+        find("div", class: "cf-select__option", text: Constants.TASK_ACTIONS.END_TIMED_HOLD.label).click
```

Comment:
Helper function for all of these if you're interested!
```suggestion
        click_dropdown(text: Constants.TASK_ACTIONS.END_TIMED_HOLD.label)
```